src/ProductBundle/DependencyInjection/SonataProductExtension.php

```diff
@@ -28,8 +28,10 @@ class SonataProductExtension extends Extension
     /**
      * Loads the product configuration.
      *
-     * @param array            $configs   An array of configuration settings
+     * @param array            $configs An array of configuration settings
      * @param ContainerBuilder $container A ContainerBuilder instance
```

Comment:
types are already defined, don't see any value in comments as the names explain well